index.Rmd

```diff
@@ -89,107 +93,101 @@ Scroll down the license with enter. Accept License (i.e. type yes). Use default
 
 ```{r, eval=FALSE, engine='bash'}
-conda create --name geotest python=2.7 numpy
+mamba create --name geotest python numpy
 ```
 
-This would create a new environment called _geotest_ with Python 2.7 and _NumPy_ installed into the conda environment. Another option is to create an environment from a `.yaml` file, in which all required modules are listed; see the example later on. To create an environment from such a file, use the argument `--file` (or `--f` in short), see the conda (documentation)[https://docs.conda.io/projects/conda/en/latest/commands/create.html]. To list the available environments:
+This would create a new environment called `geotest` with Python 2.7 and `NumPy` and `Spyder` installed into the `conda` environment. Another option is to create an environment from a `.yaml` file, in which all required modules are listed. You will see an example of this later on in this tutorial. To create an environment from such a file, you can use the argument `--file` (or `--f` in short). 
+
+Let's first list the currently available environments:
 
 ```{r, eval=FALSE, engine='bash'}
-conda info --envs
+mamba info --envs
 ```
 
-Conda puts an asterisk (*) in front of the active environment. To activate the environment:
+Mamba puts an asterisk (*) in front of the active environment. To activate the environment:
 
 ```{r, eval=FALSE, engine='bash'}
-## Cross-platform (but not always working, like in our VDI, try it)
+# Cross-platform (but not always working)
 conda activate geotest
-## Linux, macOS
+
+# Linux, macOS
 source activate geotest
-## Windows
+
+# Windows
 activate geotest
 ```
 
-After this, the current environment is shown in parentheses in front of your prompt (`(geotest)$`). Note that the activated environment is only valid for the shell in which you activated it. For instance, if you close the shell window and open a new one you will have to activate it again. If you want to install a new package and are inside the conda environment, the package is installed simply with conda or pip. Generally, installing with conda is recommended. In the documentation of most Python packages the recommended way to install can be found.
+After this, the current environment is shown in parentheses in front of your prompt (`(geotest)$`). Note that the activated environment is only valid for the shell in which you activated it. For instance, if you close the shell window and open a new one you will have to activate it again.
 
-After creating a conda environment, (additional) Python packages can be installed. There are three recommended ways to install packages, which we list below. In the documentation of most Python packages the recommended way to install them can be found.
+After creating a `conda` environment, (additional) Python packages can be installed. There are three possible ways to install packages, which we list below. In the documentation of most Python packages the recommended way to install them can be found.
```

Comment:
I would leave out the part about looking at package documentation. The documentation almost every time says to use pip, but that does not mean that it's a good idea to do that in a conda environment.